Makefile

```
@@ -89,6 +89,9 @@ help: ## Display this help.
 init:
 	git submodule init
 	git submodule update
+	helm repo add bitnami https://charts.bitnami.com/bitnami
+	helm repo add backstage https://backstage.github.io/charts
+	helm repo add janus-idp https://janus-idp.github.io/helm-backstage
```

```suggestion
	helm repo add --force-update bitnami https://charts.bitnami.com/bitnami
	helm repo add --force-update backstage https://backstage.github.io/charts
	helm repo add --force-update janus-idp https://janus-idp.github.io/helm-backstage
```

The `helm repo add` command will fail if the repository is already present. This flag overrides that behavior and sets the repository URL to what we want.